README.rst

```diff
@@ -157,6 +157,10 @@ gives a more detailed list on these.
 Building from Sources
 ---------------------
 
+.. note:: RAUC is intended to be build both as a host tool as well as a target
+   tool (service). Therfor it is fully prepared for `automake cross-compilation
```

Comment:
@UVV-gh when clicking on the commit, you can also directly add a note to this line :)

Thanks for the hint!